PATTERNS.md

```
@@ -0,0 +1,336 @@
+| Server route | `[resource].routes.ts` | `tables.routes.ts`, `records.routes.ts` |
+| Base DAO (PG) | `[action]-[resource].dao.ts` | `add-column.dao.ts` |
+| MySQL DAO | `[action]-[resource].mysql.dao.ts` | `add-column.mysql.dao.ts` |
+| MongoDB DAO | `[action]-[resource].mongo.dao.ts` | `add-column.mongo.dao.ts` |
+| MSSQL DAO | `[action]-[resource].mssql.dao.ts` | `add-column.mssql.dao.ts` |
+| Shared type | `[feature].types.ts` | `column-info.types.ts` |
+| Core type | `[feature].type.ts` | `table.type.ts` |
+| Test | `[file-name].test.ts` | `parse-bulk-data.test.ts` |
+
+## Directory Structure
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)